apps/job-service/src/jobs/dto/delayed-job-update.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsBoolean, IsUUID, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+
+class JobAttributes {
+  @IsBoolean()
+  @ApiProperty({ description: 'Value to set for isAcknowledged', example: true })
+  isAcknowledged: boolean;
+}
+
+export class JobData {
+  @ApiProperty({ enum: ['jobs'], description: 'Type of the resource', example: 'jobs' })
+  type: 'jobs';
```

Comment:
This type must match the type returned by the endpoints. In this case, it should be `delayedJobs` (make sure to update example in OpenAPI docs as well). IMO it would also make sense to rename the classes to `DelayedJob`Data/Attributes